model/src/main/kotlin/com/testerum/model/text/parts/param_meta/field/TypeMetaFieldDescriptor.kt

```diff
@@ -3,7 +3,7 @@ package com.testerum.model.text.parts.param_meta.field
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.testerum.model.text.parts.param_meta.TypeMeta
 
-data class FieldTypeMeta (
-        @JsonProperty("name") val name: String,
-        @JsonProperty("type") val type: TypeMeta
+data class TypeMetaFieldDescriptor(
```

Comment:
If you rename a model class you should rename it in the frontend too. Do you think this name is better?
I as abstrat as before for me.

Reply:
It's not a ``TypeMeta`` (it doesn't implement that interface), and you couldn't see this difference when you compare ``FieldTypeMeta`` with ``StringTypeMeta``.